pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=61.2"]
+requires = ["setuptools>=61.2", "setuptools-rust"]
```

Comment:
Maybe you should consider using [`maturin`](https://github.com/pyo3/maturin)?

Reply:
I don't think I need any of its features for the moment, and would rather avoid the extra (mandatory) dependency.